sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/DigitalTwinsRestClient.cs

```diff
@@ -15,25 +15,36 @@ namespace Azure.DigitalTwins.Core
     // objects for json inputs/return values
     internal partial class DigitalTwinsRestClient
     {
-        private const string IfMatchHeaderKey = "If-Match";
+        private const string DateTimeOffsetFormat = "MM/dd/yy H:mm:ss zzz";
 
-        internal HttpMessage CreateAddRequest(string id, string twin)
+        internal HttpMessage CreateAddRequest(string id, string twin, DigitalTwinsAddOptions digitalTwinsAddOptions)
         {
-            HttpMessage message = _pipeline.CreateMessage();
-            Request request = message.Request;
+            var message = _pipeline.CreateMessage();
```

Comment:
nit: Same here , explicit type

Reply:
See above. This is generated code copied into our customized code, so I won't be holding it to our usual code standards